docs/source/devenv-install.md

```diff
@@ -32,44 +37,45 @@ To keep your environment clean, use a virtualenv.
 
 First, create a new virtual environment called "tut" and activate it:
 ```shell
-	$ cd ~/devel
-	$ virtualenv -p /usr/bin/python2.7 tut
-	$ . tut/bin/activate
+$ cd ~/devel
+$ virtualenv -p /usr/bin/python2.7 tut
+$ . tut/bin/activate
 ```
 
 Then, install the framework by using the pip package management system:
 ```shell
-	$ pip install git+https://github.com/leapp-to/leapp
+$ pip install git+https://github.com/leapp-to/leapp
```

Comment:
Isn't that old link? :smiling_imp:

Reply:
It's being redirected, but you're right.